### ISSUE TITLE
mailfy.py returns error with specific domains

### ISSUE THREAD

**Issue by alexandr84** [i3visio/osrframework#144]
Whenever I look for an email (or emails list) that belongs to the domains of:

hotmail.com
yahoo.com
yahoo.es

I get this error:

Traceback (most recent call last):
  File "/usr/local/bin/mailfy.py", line 332, in <module>
    main(args)
  File "/usr/local/bin/mailfy.py", line 252, in main
    results = performSearch(emails, args.threads)
  File "/usr/local/bin/mailfy.py", line 193, in performSearch
    pool = Pool(nThreads)
  File "/usr/lib/python2.7/multiprocessing/__init__.py", line 232, in Pool
    return Pool(processes, initializer, initargs, maxtasksperchild)
  File "/usr/lib/python2.7/multiprocessing/pool.py", line 152, in __init__
    raise ValueError("Number of processes must be at least 1")
ValueError: Number of processes must be at least 1

I have no problem looking for @gmail.com or @hotmail.es addresses. I don't know if other domains are affected.

Thank you.

**Comment by i3visio** [comment]
We'll check that because it should not be happening. There are some platforms that won't work because the library that we are using to address this issue is not capable of performing the query correctly.

**Comment by i3visio** [comment]
Fixed in OSRFramework 0.12.1. The platforms were automatically blacklisted by the app as they cannot be safely verified. By doing so it was raising an error as it was trying to create a Pool with 0 threads which is not an option. You should be able to test it now by:
```
sudo pip install osrframework --upgrade
mailfy.py -m "whatevermailblablabla@yahoo.es"
```
